provided-template.html

```diff
@@ -0,0 +1,453 @@
+        font-family: 'Fira Sans', sans-serif;
+      }
+      .position {
+        color: white;
+        font-size: 14px;
+        font-weight: 300;
+        font-style: normal;
+      }
+      .name {
+        margin: 12px 0px 0px 0px;
+        font-weight: 400;
+        font-style: normal;
+      }
+      .events p {
+        line-height: 20px;
+      }
+      .fa {
+        font-size: 30px;
+        text-align: right;
+        padding-top:20px;
+      }
+      .social-icons {
+        color: #444444;
+        height: 36px;
+        width: 36px;
+        margin: 0em;
+        padding: 0em;
+        display: inline-block;
+        vertical-align: text-top;
+        border: 1px solid #444444;
+        text-align: center;
+        border-radius: 50%;
+        line-height: 36px;
+        font-size: 21px;
+        margin-right:25px;
+        font-weight: 600;
+        padding-top:0px;
+      }
+      .social-icons:hover {
+        color: #387C2C;
+      }
+      .social-bar {
+        text-align: center;
+      }
+      .social-bar-links {
+        text-decoration: none;
+        color: #858585;
+        margin-top:20px;
+      }
+      .social-bar-links:hover {
+        color: #387C2C;
+      }
+
+    </style>
+
+    <table align="center" border="0" cellpadding="0" cellspacing="0" width="960" bgcolor="white">
+<!-- 1th Row Header-->
+      <tr>
+        <td align="center" bgcolor="#3b7b30">
+          <a href="https://www.pausd.org/"><img src="images/pausd.svg" alt="PAUSD LOGO" width="300" height="180" style="display: block;"></a>
+          <h4 style="text-align:left;margin:-35px 0px 0px 40px;position:absolute;font-weight: 300;">NOVEMBER 2016</h4>
+        </td>
+      </tr>
+<!-- 2th Row Article -->
+      <tr>
+        <td>
+          <table border="0" cellpadding="0" cellspacing="0" width="100%" style="padding:  40px; text-align: center;">
+            <tr>
+              <!-- Left Column -->
+              <td width="40%" valign="top" style="padding-right:12px;">
+                <table border="0" cellpadding="0" cellspacing="0" width="100%">
+                  <tr>
+                    <td>
+                      <a style="text-decoration:none;"  href="" target="_blank"><h2 class="align-left">PAUSD Kindergartners to Begin New School Day Schedule</h2></a>
+                    </td>
+                  </tr>
+                  <tr>
+                    <td>
+                      <p style="margin-top:17px;line-height: 21px;" class="align-left">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries</p>
+                    </td>
+                  </tr>
+                  <tr>
+                    <td>
+                      <p class="read-more align-left"><a href="index.html">+ Read More</a></p>
```

Comment:
css classes, or inline styles as has to be the case in email templates, should be applied to the element the styles are intended for. So instead of applying a class to the `<p>` it should be applied to `<a>`. The styles:
````
      .read-more a {
        text-decoration: none;
        font-size:13px;
        font-weight: 600;
        color: #347928;
        font-family: 'Fira Sans', sans-serif;
      }
````
are intended for the `<a>` not the `<p>`.  It's a good rule to follow, and is more semantically correct, and in this case email clients won't always applied inherited styles from parent elements, and will apply there own default styles to the `<a>` tags. 

In email clients this leads to the links having the default blue colors, and text decorations.